tests/test.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-env mocha */
+/* eslint-disable func-names, prefer-arrow-callback */
+
+// These are Chimp globals
+/* globals browser */
+
+const username = 'user-test-'+Date.now();
+
+describe('Basic usage', function() {
+	before(function() {
+		browser.url('http://localhost:3000');
+	});
+
+	it('load page', function(done) {
+		browser.waitForExist('body');
+		done();
+	});
+
+	it('crate user', function(done) {
+		browser.waitForExist('.register');
+		browser.click('.register');
+
+		browser.waitForExist('[name=name]');
+
+		browser.setValue('[name=name]', username);
+		browser.setValue('[name=email]', username+'@rocket.chat');
+		browser.setValue('[name=pass]', 'rocket.chat');
+		browser.setValue('[name=confirm-pass]', 'rocket.chat');
+
+		browser.click('.submit > button');
+
+		browser.waitForExist('form#login-card input#username', 5000);
+
+		browser.click('.submit > button');
+
+		browser.waitForExist('.main-content', 5000);
+
+		done();
+	});
+
+	it('logout', function(done) {
+		browser.waitForVisible('.account-box');
+		browser.click('.account-box');
+		browser.pause(200);
```

Comment:
Yes I guess, cuz the element is moving/animating I think this was causing some test fails